pkg/client/errors_test.go

```diff
@@ -44,3 +44,19 @@ func TestPrintErrorAndExit(t *testing.T) {
 	}
 	t.Errorf("expected exit status 1, got err %v", err)
 }
+
+func TestPrintConnectionErrorAndExit(t *testing.T) {
+	if os.Getenv("PRINT_CONNECTION_ERROR_AND_EXIT") == "1" {
+		PrintConnectionErrorAndExit(
+			status.Errorf(codes.Unavailable, "Server Unavailable"),
+		)
+		return
+	}
+	cmd := exec.Command(os.Args[0], "-test.run=TestPrintConnectionErrorAndExit")
+	cmd.Env = append(os.Environ(), "PRINT_CONNECTION_ERROR_AND_EXIT=1")
+	err := cmd.Run()
+	if e, ok := err.(*exec.ExitError); ok && !e.Success() {
+		return
+	}
+	t.Errorf("expected exit status 1, got err %v", err)
+}
```

Comment:
For me the the two functions are so close the one doesn't need to test the two, but it's ok.

Reply:
The idea of testing was more for coverage points, but it seems that it did not make any difference.

So, i can remove you think is better